introduction.md

```diff
@@ -0,0 +1 @@
+Howdy, I'm a Junior web developer. Look forward to working with you all!!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.